src/styles/main.scss

```diff
@@ -1,3 +1,75 @@
+$start-day: (
+  mon 0,
+  tue 101px,
+  wed 202px,
+  thu 303px,
+  fri 404px,
+  sat 505px,
+  sun 606px
+);
+
+@mixin squareSize($sizeOfdays) {
+  width: $sizeOfdays;
+  height: $sizeOfdays;
+}
+
+%positionCenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
 body {
   margin: 0;
 }
+
+.page {
+  @extend %positionCenter;
+  min-height: 100vh;
+}
+
+.calendar {
+  min-width: 720px;
+  width: 720px;
```

Comment:
The width shouldn't be hardcoded too.
Also, you don't need a min-width if the width is fixed.